core/hypermodeling/package.json

```diff
@@ -78,4 +75,4 @@
     ],
     "extends": "plugin:@itwin/itwinjs-recommended"
```

Comment:
mind adding those rules for no-duplicate imports and type assertions like we do in VCR